lib/galaxy/files/sources/posix.py

```diff
@@ -53,6 +55,10 @@ def __init__(self, **kwd: Unpack[PosixFilesSourceProperties]):
         self.enforce_symlink_security = props.get("enforce_symlink_security", DEFAULT_ENFORCE_SYMLINK_SECURITY)
         self.delete_on_realize = props.get("delete_on_realize", DEFAULT_DELETE_ON_REALIZE)
         self.allow_subdir_creation = props.get("allow_subdir_creation", DEFAULT_ALLOW_SUBDIR_CREATION)
+        self.prefer_symlinks = props.get("prefer_links", DEFAULT_PREFER_LINKS)
```

Comment:
Can we just name this `prefer_links` for consistency?

Reply:
Let me know if this is better. There is a conflicting method name but I think this should work.